Lib/test/test_types.py

```diff
@@ -1360,6 +1360,20 @@ class C: pass
         D = types.new_class('D', (A(), C, B()), {})
         self.assertEqual(D.__bases__, (A1, A2, A3, C, B1, B2))
 
+    def test_get_orig_bases(self):
+        T = typing.TypeVar('T')
+        class A: pass
+        class B(object): pass
```

Comment:
What purpose does this class serve? Inheriting directly from `object` is always redundant on Python 3